### ISSUE TITLE
Running jsonlint with patterns in configfile blocks on stdin since v14.0.1

### ISSUE THREAD

**Issue by hdcore** [prantlf/jsonlint#18]
I'm working with only configuationfiles in my projects instead of cmdline parameters. But since version 14.0.1 it don't work anymore because he now always waits for stdin input.

Example of a config file: `.jsonlintrc.yaml`:
```
---
patterns:
  - "**/*.json"
  - "!resources/**"
  - "!themes/**"
continue: true
log-files: true
```
It worked before <V14 with only `jsonlint -ln`. Now I must always give arguments like the name, the directory or a pattern. But in that case he is ignoring the patterns values in the config file.

This looks caused by the fix in #17.

**Comment by prantlf** [comment]
Oh, including #18 in a pushed commit closed the issue automatically!

Thanks for reporting this and for a clear description with an example! I missed the usage of the patterns property, I hope I'll get to improving the test code coverage.

The problem is fixed in [14.0.2](https://github.com/prantlf/jsonlint/releases/tag/v14.0.2). I'm sorry for the trouble.

**Comment by hdcore** [comment]
Thanks it works again. The configuration file was the main reason I've changed to this fork.
